File: solutions/3331-minimum-operations-to-exceed-threshold-value-i/solution.js
var minOperations = function(nums, k) {
  nums.sort((a, b) => a - b);
  let count = 0;
  for (let i = 0; i < nums.length; i++) {
    if (nums[i] >= k) {
      break;
    }
    count++;
  }
  return count;
};

File: solutions/3331-minimum-operations-to-exceed-threshold-value-i/solution.java
class Solution {
    public int minOperations(int[] nums, int k) {
        Arrays.sort(nums);
        int count=0;
        for(int i=0;i<nums.length;i++){
            if(nums[i]>=k){
                break;
            }
            count+=1;
        }
        return count;
    }
}
